src/toTransform.js

```diff
@@ -39,36 +39,77 @@
 
   function convertPath (properties) {
     var offsetPath = internalScope.offsetPathParse(properties['offset-path']);
-    if (!offsetPath || offsetPath.type !== 'path') {
+
+    if (!offsetPath) {
       return null;
     }
 
+    if (offsetPath.type === 'path') {
+      return convertPathString(properties);
+    }
+
+    if (offsetPath.type === 'ray') {
+      return convertRayString(properties);
+    }
+  }
+
+  function getOffsetDistanceLength (offsetDistance, pathElement) {
+    if (offsetDistance.unit === '%') {
+      return Number(offsetDistance.value) * pathElement.getTotalLength() / 100;
+    } else {
+      return Number(offsetDistance.value);
+    }
+  }
+
+  function convertPathString (properties) {
+    var offsetPath = internalScope.offsetPathParse(properties['offset-path']);
+
     var offsetDistance = internalScope.offsetDistanceParse(properties['offset-distance']);
     if (offsetDistance === undefined) {
       offsetDistance = {value: 0, unit: 'px'};
     }
 
     pathElement.setAttribute('d', offsetPath.input);
 
-    var offsetDistanceLength;
-    if (offsetDistance.unit === '%') {
-      offsetDistanceLength = Number(offsetDistance.value) * pathElement.getTotalLength() / 100;
-    } else {
-      offsetDistanceLength = Number(offsetDistance.value);
-    }
+    var offsetDistanceLength = getOffsetDistanceLength(offsetDistance, pathElement);
 
     var point = pathElement.getPointAtLength(offsetDistanceLength);
 
     return 'translate3d(' + point.x + 'px, ' + point.y + 'px, 0px)';
   }
 
+  function convertRayString (properties) {
+    var offsetPath = internalScope.offsetPathParse(properties['offset-path']);
+
+    var offsetDistance = internalScope.offsetDistanceParse(properties['offset-distance']);
+    if (offsetDistance === undefined) {
+      offsetDistance = {value: 0, unit: 'px'};
+    }
+
+    pathElement.setAttribute('d', offsetPath.input);
```

Comment:
We shouldn't be setting d. Perhaps getOffsetDistanceLength should take a path length as input. We can pass in 0 for ray, with a FIXME to calculate path length based on <size>.

For now, all % lengths will become 0 for ray.